### DISCUSSION TITLE
How jxls:SimpleConfigurator support latest logback version?

### DISCUSSION THREAD

**Discussion by RayWang0330** [jxlsteam/jxls#232] [Q&A] [answered]
Hi ,
`SimpleConfigurator` extend logback class for version : **1.2.10**, but the latest logback version(**1.4.x** for Java11) update class `GenericConfigurator` to `GenericXMLConfigurator`. Is there fix planning to update this class?
Thanks!

**Comment by leonate** [comment, marked as answer]
Hi @RayWang0330,

No plans at the moment. Feel free to submit a pull request.
